src/mono/System.Private.CoreLib/src/System/Reflection/Emit/DerivedTypes.Mono.cs

```diff
@@ -453,22 +453,22 @@ protected override bool IsByRefImpl()
 
         public override Type MakeArrayType()
         {
-            throw new ArgumentException("Cannot create an array type of a byref type");
+            throw new ArgumentException(SR.Argument_CannotCreateByrefTypeOfExistingByrefType);
         }
 
         public override Type MakeArrayType(int rank)
         {
-            throw new ArgumentException("Cannot create an array type of a byref type");
+            throw new ArgumentException(SR.Argument_CannotCreateByrefTypeOfExistingByrefType);
         }
 
         public override Type MakeByRefType()
         {
-            throw new ArgumentException("Cannot create a byref type of an already byref type");
+            throw new ArgumentException(SR.Argument_CannotCreateByrefTypeOfExistingByrefType);
         }
```

Comment:
Let me try another way. Lines 456 and 461 both have the same message - "Cannot create an array type of a byref type". Line 466 has a different message - "Cannot create a byref type of an already byref type". This would imply they can't all have the same resource string, that was my point. In the previous commit all of them had the same messages from 456/461. Now all of them have the same message from 466. I think the first two should have the previous resource string and the last one should have the current resource string.

Reply:
Apologies, must get my eyes checked [😂](https://emojipedia.org/face-with-tears-of-joy/)
You mean like this?

`
public override Type MakeArrayType()
        {
            throw new ArgumentException(SR.Argument_CannotCreateArrayOfByrefType);
        }

        public override Type MakeArrayType(int rank)
        {
            throw new ArgumentException(SR.Argument_CannotCreateArrayOfByrefType);
        }

        public override Type MakeByRefType()
        {
            throw new ArgumentException(SR.Argument_CannotCreateByrefTypeOfExistingByrefType);
        }

        public override Type MakePointerType()
        {
            throw new ArgumentException(SR.Argument_CannotCreatePointerTypeOfByref);
        }`

Reply:
W00t! Yep. That would make sense to me.